pkg/adapter/nimbus-kyverno/clusterrole.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/component: background-controller
+    app.kubernetes.io/instance: kyverno
+    app.kubernetes.io/part-of: kyverno
+  name: kyverno:update-pods
+rules:
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
```

Comment:
why do we need all resources ? Can we not restrict to only those required ? Deployments only ?